### ISSUE TITLE
Quick toggle tile or widget for start/stop of capture in pcapdroid

### ISSUE THREAD

**Issue by K-techie** [emanuele-f/PCAPdroid#227]
A quick toggle tile in the notification panel would be a handy way to start/stop the capture in pcapdroid. Its significance increases with the Firewall feature being implemented as it would be easy for people to toggle firewall on or off by controlling the capture through quick settings bar.
Also , when always-on vpn (with block connection without VPN)  is toggled on , it makes sense to just start/stop the capture to cut-off internet connection instead of using the internet toggle to do so, as the apps won't keep on trying to connect in the background and use more battery.
In short this shortcut would be useful to be used along with the firewall feature.
